SQMs/missionTanoa.sqm

```diff
@@ -51137,5 +51139,208 @@ class Mission
 			colorName="ColorWEST";
 			id=5759;
 		};
+		class Item1126
+		{
+			dataType="Object";
+			class PositionInfo
+			{
+				position[]={7373.3325,9.2331867,8418.5234};
+				angles[]={0,5.5739436,0};
+			};
+			side="Empty";
+			flags=5;
+			class Attributes
+			{
+			};
+			id=5761;
+			type="Land_Billboard_F";
+		};
+		class Item1127
+		{
+			dataType="Group";
+			side="West";
+			class Entities
+			{
+				items=1;
+				class Item0
+				{
+					dataType="Object";
+					class PositionInfo
+					{
+						position[]={5631.0811,3.6714213,10142.964};
+					};
+					side="West";
+					flags=7;
+					class Attributes
+					{
+						isPlayer=1;
+					};
+					id=5775;
+					type="B_soldier_AAA_F";
```

Comment:
Why is there an [Asst. Missile Specialist](https://community.bistudio.com/wiki/Arma_3_CfgVehicles_WEST#B_soldier_AAA_F)?